pkg/operator/starter.go

```diff
@@ -316,6 +323,7 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 		configInformers.Config().V1().Networks(),
 		etcdClient,
 		controllerContext.EventRecorder,
+		&etcdmemberlistercontroller,
```

Comment:
this line does not compile, i do not know how to fix it :/ 

So the `NewClusterMemberController` returns a `factory.Controller` type . I need to have a ref for the `etcdmemberlistercontroller` to update the local cache in each controller upon notifications 

please let me know wdyt

Reply:
I suggest you to create an interface that is returned by `NewEtcdMemberListerController` instead. You're also missing to actually run that controller.

Reply:
fixed 👍🏽